packages/cardhost/app/components/save-button.js

```diff
@@ -13,7 +18,16 @@ export default class SaveButton extends Component {
 
   @tracked justSaved;
 
-  @task(function*() {
+  autosaveDebounce = autosaveDebounce || AUTOSAVE_DEBOUNCE;
+  autosaveDisabled = typeof this.args.autosaveDisabled === 'boolean' ? this.args.autosaveDisabled : !!autosaveDisabled;
+
+  get cardIsNew() {
+    let card = this.args.card;
+    return card.isNew;
+  }
+
+  @enqueueTask
```

Comment:
I think maybe we want `[@keepLatest`](http://ember-concurrency.com/api/TaskProperty.html) instead of `enqueue`. A GitHub roundtrip is 4 seconds, and so I think bursts of typing or whatever could rapidly spawn 20+ seconds of queued requests. Is that accurate?

Reply:
@jenweber  the saves need to be enqueued so that we don't have `/meta/version` mismatches. We could mitigate the scenario you described by making the debounce longer (maybe 3-5 seconds?)

I'll put some comments in the code to reflect this.